docs/monitoring/model-monitoring-deployment.ipynb

```diff
@@ -53,7 +53,7 @@
     "* **Total Variation Distance** (TVD) &mdash; The statistical difference between the actual predictions and the model's trained predictions.\n",
     "* **Hellinger Distance** &mdash; A type of f-divergence that quantifies the similarity between the actual predictions, and the model's trained predictions.\n",
     "* **Kullback–Leibler Divergence** (KLD) &mdash; The measure of how the probability distribution of actual predictions is different from the second model's trained reference probability distribution.\n",
-    "* **Model Endpoint** &mdash; A combination of a deployed Nuclio function and the models themselves. One function can run multiple endpoints; however, statistics are saved per endpoint.\n",
+    "* **Model Endpoint** &mdash; A combination of a model and a runtime function which can be a deployed Nuclio function or a job runtime. One function can run multiple endpoints; however, statistics are saved per endpoint.\n",
```

Comment:
"* **Model Endpoint** &mdash; A combination of a model and a runtime function that can be a deployed Nuclio function or a job runtime. One function can run multiple endpoints; however, statistics are saved per endpoint.\n",